src/main/java/org/example/services/AuthService.java

```diff
@@ -0,0 +1,61 @@
+package org.example.services;
+
+import io.jsonwebtoken.Jwts;
+import org.example.daos.AuthDao;
+import org.example.exceptions.Entity;
+import org.example.exceptions.FailedToCreateException;
+import org.example.exceptions.InvalidException;
+import org.example.models.LoginRequest;
+import org.example.models.RegisterRequest;
+import org.example.models.User;
+
+import java.security.Key;
+import java.sql.Date;
+import java.sql.SQLException;
+
+public class AuthService {
+    final AuthDao authDao;
+    final Key key;
+
+    final int tokenLifetime = 28800000;
```

Comment:
env this

Reply:
Pretty pretty please with a cherry on top allow me to get away with this :)))

Also they arguably did it in a worse way than this: https://github.com/Iced-Linta/il-ems-dropwizard/blob/8c30e009ac28f82d6eea3620337849e42a8c4608/src/main/java/org/example/services/AuthService.java#L47